_posts/wiki/2018-10-18-ROS-kinetic-installation.md

```diff
@@ -73,15 +73,16 @@ Now, there are a number of options to choose from. The recommended one is the _D
 
 ~~~bash
 rosinstall_generator desktop_full --rosdistro kinetic --deps --wet-only --tar > kinetic-desktop-full-wet.rosinstall
-wstool init -j8 src kinetic-desktop-full-wet.ros
+wstool init -j8 src kinetic-desktop-full-wet.rosinstall
 ~~~
 
 ### Resolving dependencies
 
-Before you can build your catkin workspace, you need to make sure that you have all the required dependencies. They use the `rosdep` tool for this. NOTE: some of the packages might be unavailable and may need to be installed seperately.
+Before you can build your catkin workspace, you need to make sure that you have all the required dependencies. They use the `rosdep` tool for this. NOTE: some of the packages might be unavailable and may need to be installed seperately. A useful command for finding packages is "apt-cache search ros-kinetic" in conjunction with "sudo apt-get install ros-kinetic-PACKAGE".
```

Comment:
Nice that you added this. However, two problems: 

 * code tags should not come with double quotes, but with `
 * You are not explicitly saying what is the command doing and why it is useful to do it. I would add something on this.